### DISCUSSION TITLE
Installing Zeek after initial SO Standalone installation

### DISCUSSION THREAD

**Discussion by revast** [Security-Onion-Solutions/securityonion#6345] [Unsupported Versions] [answered]
Hello, 

We did a standalone Install of SO off Ubuntu18.04, and selected Suricata at install wizard, instead of Zeek. Now we would like to have Zeek activated, is there a possibility to do so afterwards? 

From what we grasped so far, there is no simple "enable zeek /Switch from Suricata to Zeek" option - it seems more feasible to reinstall SO again...

Which would be also ok, we just wanted to know if there is a way to enable it / switch these components.

**Comment by dougburks** [comment, marked as answer]
Have you tried the steps in the documentation but changing `SURICATA` to `ZEEK`?
https://docs.securityonion.net/en/2.3/suricata.html#metadata

**Reply by kornflake003-GITHUB** [reply]
Hello,

In our case we also have a "standalone" installation and during the initial configuration we selected SURICARA as the METADATA and IDS processor. And we follow the recommendation of @dougburks  changing SURICATA for SEEK in the file "global.sls". And then running the following commands:

- sudo salt '*' cmd.run 'so-suricata-stop'
- sudo salt '*' state.highstate

It worked correctly. The change was reflected in the file "suricata.yaml".

**Reply by revast** [reply]
nice, thanks!
